lapis2/src/test/kotlin/org/genspectrum/lapis/model/VariantQueryFacadeTest.kt

```diff
@@ -23,7 +26,7 @@ class VariantQueryFacadeTest {
 
     @Test
     fun `given a complex variant query then map should return the corresponding SiloQuery`() {
-        val variantQuery = "300G & (400- | 500B) & !600 & MAYBE(700B | 800-) & [3-of: 123A, 234T, 345G] & A.1.2.3*"
+        val variantQuery = "300G & (400- | 500B) & !600. & MAYBE(700B | 800-) & [3-of: 123A, 234T, 345G] & A.1.2.3*"
```

Comment:
600 should be also a valid variant query. Why was this changed here?

Reply:
Yes, but it doesn't mean `NucleotideSymbolEquals(null, 600, "-")` anymore. It's translated to a `HasNucleotideMutation`. I changed it such that I don't have to modify the output by much. But actually it doesn't matter. We can also keep the original version here and expect a `HasNucleotideMutation`